tasks/Resources/Provisioning/Hive/AWS/oc-provision.robot

```diff
@@ -0,0 +1,56 @@
+*** Settings ***
+Library           OperatingSystem
+
+*** Keywords ***
+Run AWS Configuration In Hive
+    [Arguments]    &{infrastructure_configurations}
+    Oc Apply    kind=List    src=tasks/Resources/Provisioning/Hive/AWS/aws-cluster.yaml
+    ...    template_data=${infrastructure_configurations}
+    Sleep    60
+    ${namespace} =    Oc Get    kind=Namespace    label_selector=hive.openshift.io/cluster-pool-name=${infrastructure_configurations}[hive_cluster_name]
+    Log    ${namespace[0]['metadata']['name']}    console=True
+    ${pod} =    Oc Get    kind=Pod    namespace=${namespace[0]['metadata']['name']}
+    Log    ${pod[0]['metadata']['name']}    console=True
+    Wait Until Keyword Succeeds    40 min    10 s
+    ...    Wait Until Cluster Is Provisioned    ${pod[0]['metadata']['name']}    ${namespace[0]['metadata']['name']}
+
+Wait Until Cluster Is Provisioned
+    [Arguments]    ${pod_name}    ${namespace}
+    ${installation_log} =    Oc Get Pod Logs    name=${pod_name}    container=hive    namespace=${namespace}
+    Should Contain    ${installation_log}    install completed successfully
+
+Claim Cluster
+    Oc Apply    kind=ClusterClaim    src=tasks/Resources/Provisioning/Hive/AWS/claim.yaml
+    ...    template_data=${infrastructure_configurations}
+    Wait Until Keyword Succeeds    15 min    10 s
+    ...    Wait Until Cluster to be Claimed
+
+Wait Until Cluster to be claimed
+    ${status} =    Oc Get    kind=ClusterClaim    name=${infrastructure_configurations}[hive_claim_name]    namespace=rhods
+    Should Be Equal As Strings    ${status[0]['status']['conditions'][0]['reason']}    ClusterClaimed
+
+Login To AWS Cluster With Hive
+    Set Log Level    None
+    ${ns} =    Oc Get    kind=Namespace    label_selector=hive.openshift.io/cluster-pool-name=${infrastructure_configurations}[hive_cluster_name]
```

Comment:
I think ${infrastructure_configurations} should be a parameter or even better just parameter ${cluster_name}